app/controllers/application.js

```diff
@@ -4,7 +4,7 @@ const { Controller, computed, inject, get } = Ember;
 
 export default Controller.extend({
   session: inject.service(),
-
+  currentUser: computed.oneWay('session.session.content.authenticated'),
```

Comment:
feels a bit law of demeter-y violate-y. i think it's ok for now but i'd probably (in a later PR) refactor to move currentUser/session and isAuthenticated into its own "auth service" or something.